genshinstats/genshinstats.py

```diff
@@ -52,6 +52,17 @@ def set_cookie_header(header: str) -> None:
     c.load(header)
     session.cookies.update(c)
 
+def get_browser_cookie(browser: str=None) -> dict:
+    """Get cookies from browser"""
+    import browser_cookie3
+    load = getattr(browser_cookie3,browser.lower()) if browser else browser_cookie3.load
+    # we want to keep user data secure, so don't use any login ticket cookies
+    cookie={}
+    for c in load(domain_name='.mihoyo'):
+        if c.name in ('ltuid','ltoken'):
+            cookie.update({c.name:c.value})
+    return cookie
```

Comment:
Could you make this be a dict comp? I think it would look nicer like that.
We could also return a new cookiejar object, the original for loop was just to get rid of the domain and expire attributes.
It'd be nice if you added a similar docstring like in `set_cookie_auto` since the API docs are generated from those and I'd like to make them clear as to what exactly each parameter does.

Also just for formatting sake use a space in variable assignment if you happen to use it.

Reply:
Nice advices, i'll leave the rest to you
Don't really know what to put in docstring